cli.md

```diff
@@ -42,8 +42,8 @@ URL of parameter file in JSON format
 
 ### `-l, --log-level LOGLEVEL`
 
-Minimum Log level. One of `OFF`, `ERROR`, `WARN`, `INFO` (default), `DEBUG`, `TRACE`).
-:warning: This specification overwrites all other specifications. For a fine-grained change, please use the parameter file.
+Set the global minimum Log level. One of `OFF`, `ERROR`, `WARN`, `INFO` (default), `DEBUG`, `TRACE`).
```

Comment:
Sorry, this may come too late, but I think it should really read `maximum` instead of minimum here. (Since each level has to let lower levels through, respectively, and the usual semantics of this is "how much information to display".)

Reply:
Yes, that would be clear, too. (Though I believe "log level" usually has more of a "verbosity" than a "severity" connotation.)

Reply:
I found this in the documentation of Log4J:
https://logging.apache.org/log4j/2.x/manual/customloglevels.html
Off (0), FATAL(100), ERROR (200),...
--> Set the global maximum verbosity level. More verbose log entries will be ignored.

Reply:
Fixed in a88c3ac